app/views/layouts/application.html.erb

```diff
@@ -42,6 +42,8 @@
     <% unless @do_not_show_breadcrumbs %>
       <% if @content_item.try(:back_link) %>
         <%= render 'govuk_publishing_components/components/back_link', href: @content_item.back_link %>
+      <% elsif @content_item.brexit_hub_page? %>
+        <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: [ { url: "/", title: "Home" } , { url: "/transition", title: "Brexit" } ] %>
```

Comment:
Once we have updated the contextual nav in publishing components this hardcoding can be removed.